### PR TITLE
Make botnames more better and realistic

### PR DESCRIPTION
**Pull request by AlexHGaming**

### CODE REVIEW

**Comment by AlexHGaming** [comment]
I will be cleaning the entire botnames.txt (not all the names) but remove the ones that don't really fit into a player's name and make more realistic ones up 👍

**Comment by Luka967** [comment]
>sees commit

Garbage pr close immediately

>will be

Ok gotchu

File: src/ai/BotNames.txt [hunk 1]
```diff
@@ -669,7 +667,6 @@
 <r>҉Fiz҉z҉y🚽ᴳᵒᵈ
 <r>๖ۣۜℓuca∞༻一๖大❥❥
 <r>Chinese
-<r>PENIS EATER
```
Comment (MastaCoder) [inline src/ai/BotNames.txt]: now uhm who added this xD

Comment (Megabyte918) [inline src/ai/BotNames.txt]: I used a copy-paste names extension then copied every name from like 30 party servers and pasted them into this file, I guess some bad ones made it in there too xD

Comment (Jhair2015) [inline src/ai/BotNames.txt]: (╭☞ ͡ ͡° ͜ ʖ ͡͡°)╭☞

Comment (AlexHGaming) [inline src/ai/BotNames.txt]: lmao

**Comment by Agarian** [comment]
Will somebody merge this?

**Comment by MastaCoder** [comment]
Awaiting approval of OP. 

> I **will** be cleaning the entire botnames.txt

**Comment by AlexHGaming** [comment]
well u can merge it now
@MastaCoder and delete them 2 'playercell.js' commits, it was meant to go in the other PR lol, i dunno what to do

**Comment by Agarian** [comment]
What exactly does `return(this)` do?

**Comment by MastaCoder** [comment]
@Agarian Gonna guess it returns the value of this to whatever function calling it. However, I don't really see why they'd need to do that..

**Comment by Agarian** [comment]
That's why I asked

**Comment by Agarian** [comment]
And just curious, what is the protocol version of http://old.ogarul.io ? Because just maybe it'll give me an idea on how to run Ogar-Unlimited without it, since that site is going down soon. There was a client that could be used to connect and play on Ogar-Unlimited servers (https://github.com/ItsVoid/agario-client), however, that client seems to be bricked, as it does not load anymore (likely due to cross origin issues).

**Comment by Tyler3D** [comment]
4. But you could probably replace the packet folder with some adjustments to make ogarul compatible with other protocols

**Comment by Tyler3D** [comment]
@Agarian

**Comment by Agarian** [comment]
True, however that's probably be MUCH more work than just finding another protocol 4
I may be wrong though

**Comment by Luka967** [comment]
The old Cigar used protocol 4.

**Comment by Agarian** [comment]
Noted, thanks for the info, now all i need to do is find the version lol

**Comment by Luka967** [comment]
xD

**Comment by AlexHGaming** [comment]
making new one

**Pull request closed without merging**